python_ta/reporters/templates/stylesheet.css

```diff
@@ -176,21 +178,13 @@ section {
     margin-left: auto;
 }
 
-.slider button {
-    cursor: pointer;
-    background: #427fed;
-    border: 1px solid transparent;
-    border-radius: 2px;
-    padding: 4px 12px;
-    min-width: 54px;
-    margin-left: 16px;
-    color: #fff;
-    font-weight: bold;
-    -webkit-font-smoothing: antialiased;
-    font-size: 12px;
+.slider svg:hover {
+    filter: brightness(120%);
 }
-.slider button:hover {
-    text-decoration: underline;
+
+svg.collapsed {
+    transform-origin: center;
+    transform: rotateX(180deg) translateY(-6px);
```

Comment:
Rather than use `rotate`, use `scaleY(-1)` to perform a vertical reflection. You shouldn't need the `translateY` either, if you've set up the viewbox correctly.